docs/specifications/ucs/layout/change_panel_structure/UCS_change_panel_structure.md

```diff
@@ -27,7 +27,7 @@ dashup - Use Case Specification: Layout - Change Panel Structure
 
 # 1. Change Panel Structure - Brief Description
 In the use case _change panel structure_ the platform should enable the user to create, modify and delete sections and 
-fill these sections with the desired panels. A rearrangement of sections as well as panels should be possible by drag 
+fill these sections with the desired panelspanels. A rearrangement of sections as well as panels should be possible by drag 
```

Comment:
Use refactoring carefully. This might happen as well. Please change.

Reply:
I'm asking myself, what went wrong with the auto refactoring here.